src/plugins/console/public/application/containers/remote/console_header.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const ConsoleHeader = ({
+  isConsoleOpen,
+  rightSideItem,
+  onClick,
+}: {
+  isConsoleOpen: boolean;
+  rightSideItem?: React.ReactNode;
+  onClick?: () => void;
+}) => (
+  <EuiPanel borderRadius="none" onClick={onClick} grow>
+    <EuiFlexGroup justifyContent="spaceBetween">
+      <EuiFlexItem grow={false}>
+        <EuiIcon type={isConsoleOpen ? 'arrowUp' : 'arrowDown'} size="l" />
```

Comment:
nit: I think the header visually looks mostly like an accordion and functionally close to it as well, so I'd suggest to use an "arrowRight" icon when it's closed and an "arrowDown" when it's opened.

Reply:
@julianrosado thoughts?

@yuliacech I'm following Julians [figma](https://www.figma.com/file/E5f59L6OABiEH2rb7qTKPd/Console-Flyout?type=design&node-id=243-103759&mode=design&t=eMjmva2YXyHCpnmh-0)

Reply:
Thanks for flagging Rodney and Yulia! We don't have any hard or fast rules with arrows (I have asked!) so it's a bit _subjective_.

IMO accordions are headers that the content (visually) drops down and is pulled back up into (so right arrow, down arrow make sense, as the context is the header). The header doesn't move.

The dev console is closer to a flyout, where the whole panel (header and all, not just the content) slides up and down in the context of the page. I think arrowUp / arrowDown makes that more clear to the user.